docs/content/DataSourceMetadataQuery.md

```diff
@@ -0,0 +1,31 @@
+---
+layout: doc_page
+---
+# Data Source Metadata Queries
+Data Source Metadata queries return ingestion related information for a dataSource. It returns the timestamp of latest ingested event for the datasource. The grammar is:
+
+```json
+{
+    "queryType" : "dataSourceMetadata",
+    "dataSource": "sample_datasource",
+}
+```
+
+There are 2 main parts to a Data Source Metadata query:
+
+|property|description|required?|
+|--------|-----------|---------|
+|queryType|This String should always be "dataSourceMetadata"; this is the first thing Druid looks at to figure out how to interpret the query|yes|
+|dataSource|A String defining the data source to query, very similar to a table in a relational database|yes|
```

Comment:
yap, it can be an object technically... for docs string is consistent with other query types.